config/cars/kunos/ks_porsche_718_boxster_s.ini

```diff
@@ -35,9 +60,24 @@ Resolution = 1024, 512
 @ = MultiItem, Role = DashWarningTyrePressure, Start = "231, 165", Size = "48, 38"
 @ = MultiItem, Role = HAZARD, Start = "158, 246", Size = "52, 33"
 
+; set screen to smDigitalScreen
+[SHADER_REPLACEMENT_...]
+MATERIALS = INT_DETAILS_Emissive_Ref
+MESHES = GEO_INT_DISPLAYS
+SHADER=smDigitalScreen
+PROP_... = smScreenScale, 1024, 512
+PROP_... = smScreenType, 1
+PROP_... = smUseTextureCoordinates, 1
+PROP_... = ksDiffuse, 0
+
+
+; im not quite sure what the comment below references, everything looks fine while ingame, keeping it here in case its important.
 ; TODO: fix scale to avoid having a massive texture?
```

Comment:
I don't quite understand what this references, the resulting Nav looked perfectly fine:
![image](https://user-images.githubusercontent.com/13604413/132219605-4254fb5f-a83c-4a9c-864d-95fbb67bfb99.png)

i've commented it back in for that reason